src/release/breaking-changes/trackpad-gestures.md

```diff
@@ -303,16 +303,16 @@ class Foo extends StatelessWidget {
 ```
 
 Please note: Use of raw `Listener` in this way could cause conflicts with other
```

Comment:
**chore:** Per [GSG](https://developers.google.com/style/tone#politeness-and-use-of-please), remove `Please`.

**question:** Should this be in a note block?